docs/content/node-operation/index.mdx

```diff
@@ -1,21 +1,45 @@
 ---
-title: Hello Node Operator!
+title: Flow Nodes
 sidebar_title: Introduction
-description: How to run your own Flow node
+description: Introduction Flow Nodes
 ---
 
-To run a node in the Flow network, the node must:
-* be registered with sufficient stake
-* be authorized by the service account
-
-Before proceeding, ensure you have the stake required for your new node and that your node will be authorized by the service account (apply [here](https://www.onflow.org/node-validators)).
-
-To set up a new Flow node you will need to complete the following steps:
-
-1. [Provision](/node-operation/node-setup) the machine on which your node will run.
```

Comment:
Is this being moved over to somewhere else?

Reply:
Yes, `/nodes/node-operation/node-setup`